### DISCUSSION TITLE
Assistance is required.

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#1558] [Announcements]
Could you please assist me in identifying which LXC containers are compatible with Debian 12 as the default operating system? 
If you happen to create an application LXC using Debian 12, kindly share your results in this discussion. I appreciate your help!

Please take note that if you are currently using Proxmox VE 7, refrain from creating Debian 12 LXCs due to differences in locale parameters.

**Comment by tteck** [comment]
These are the ones that have been tested up to now and have been set to default to Debian 12.

Adguard
AudioBookShelf
Bazarr
Cloudflared
Dashy
deCONZ
Docker
EMQX
ESPHome
Fhem
grocy
Home Assistant Core
Home Assistant
Homepage
Keycloak
Lidarr
MariaDB
MQTT
Nginx Proxy Manager
n8n
Node Red
Omada
Openobserve
Paperless-ngx
Photoprism
Pi-hole
PostgreSQL
qbittorrent
Prowlarr
Readarr
Radarr
Sabnzbd
Scrypted
Sonarr
Tautulli
Technitium DNS
Trilium
Unifi
Uptime Kuma
Zigbee2MQTT

**Reply by tteck** [reply]
~Paperless-ngx LXC, 
Debian 12 includes the distribution of Python 3.11. Unfortunately, a few dependencies can then no longer be resolved.~

After creating a new Debian 11 Paperless-ngx LXC, I updated it to Debian 12 which broke it, but after updating Paperless-ngx LXC via the script, it works again (it needed to migrate).

**Reply by MoldyTaint** [reply]
Do we just install them and see if they work? I'm not to technical but if it's just a simple install and wing it, I can dedicate an hour or two testing some!

**Reply by tteck** [reply]
That is the only information I require—whether they are compatible with Debian 12.

**Reply by ianalexis** [reply]
YunoHost must stay 11 until YunoHost12 is launched.
https://forum.yunohost.org/t/debian-bookworm-12-x-has-been-released-yunohost-is-not-yet-compatible/25082

**Comment by towerhand** [comment]
Plex, tautulli, radarr, readarr, bazarr, prowlarr, and sabnzbd seem to be working under Debian 12.

**Reply by tteck** [reply]
Fresh installs or updated LXCs?

**Reply by towerhand** [reply]
Fresh installs. I migrated from a 250GB hard drive to a 1TB over the weekend, so I decided to move those services to their own LXC containers instead of continuing to have them on docker.

edit: I didn't try sonarr since I'm waiting for v4.

**Reply by tteck** [reply]
Perfect, thank you!

**Reply by towerhand** [reply]
Btw, qbittorrent seems to be working fine too, but I haven’t used it much because I haven’t figured out how to migrate all my torrents and config from docker.

**Reply by towerhand** [reply]
Lidarr works too.

**Comment by alexdelprete** [comment]
Hi @tteck,

I upgraded the following LXCs, I'll put a note in () next to the ones with a warning:

- docker-ce
- emqx (works, but updating source list throws an error. packagecloud.io does not yet support bookworm probably)
- influxdb v1 (works, but updating source list throws an error. repo does not yet support bookworm probably)
- influxdb v2 (works, but updating source list throws an error. repo does not yet support bookworm probably)
- jellyfin
- mariadb
- node-red
- postgresql

**Reply by tteck** [reply]
Node-Red shouldn't have a source list

**Reply by alexdelprete** [reply]
I have this. Probably I updated something since I installed it and it created it...

```
❯ cat /etc/apt/sources.list.d/nodesource.list
deb [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_14.x bookworm main
deb-src [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_14.x bookworm main
```

Anyway, update works now...so I edited the previous post's note for node-red.

**Reply by tteck** [reply]
I stand corrected, you are correct.

**Reply by alexdelprete** [reply]
No problem. I think a couple of months ago I had a problem because of really old versions of npm/corepack/node.js, so I tinkered with npm and other stuff to upgrade everything to latest.

**Comment by oOStroudyOo** [comment]
I've not had any problems yet on Debian 12.

I've only built Lidarr and PiHole LXCs with your scripts recently but the apps I'm running are:

- **Docker with Compose, Portainer and Watchtower on 5 containers**
- **Radarr**
- **Plex Media Server**
- **Prowlarr**
- **Sonarr**
- **Trilium**
- **Lidarr**
- **AudioBookShelf**
- **Readarr**
- **PiHole**
- **Home Assistant Container**
- **File Browser**

- Cloudflared (On 16 containers)
- Ombi
- Homepage
- qBittorrent
- FlareSolverr

Apps in bold are ones that were installed via your scripts. PiHole and Lidarr was installed after Debian 12 was added to your scripts, the rest was before and I've manually changed the sources.

**Comment by samydoesit** [comment]
Fresh install of FHEM LXC on Debian 12 works without problems so far.

**Comment by jsrol** [comment]
Wiki.js works on debian12 using your script (fresh install)

**Comment by sendmebits** [comment]
Is the below still something we need to watch out for?  I'm running Proxmox VE 7.4-17.

> Please take note that if you are currently using Proxmox VE 7, refrain from creating Debian 12 LXCs due to differences in locale parameters.

**Comment by Smile2Grow** [comment]
Docker and KeyCloak are working on debian12 using your script (fresh install)

**Comment by Smile2Grow** [comment]
Wireguard only with debian11
